Tests/ClientTests/Autofill/CreditCard/ViewComponents/SensitiveHostingControllerTests.swift

```diff
@@ -0,0 +1,50 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import SwiftUI
+import UIKit
+import XCTest
+@testable import Client
+
+class SensitiveHostingControllerTests: XCTestCaseRootViewController {
+    var mockNotificationCenter: SpyNotificationCenter!
+    var mockAppAuthenticator: AppAuthenticationProtocol!
+
+    override func setUp() {
+        super.setUp()
+
+        mockNotificationCenter = SpyNotificationCenter()
+        mockAppAuthenticator = MockAppAuthenticator()
+    }
+
+    override func tearDown() {
+        super.tearDown()
```

Comment:
Ah since I'm here. Every tests create individual instance and doesn't deinit it after completion. With a tearDown we can ensure we properly clear any instances and start with a clean slate on each test case.
```suggestion
        super.tearDown()

        mockNotificationCenter = nil
        mockAppAuthenticator = nil
```